src/ContainerDetails.jsx

```diff
@@ -27,44 +27,50 @@ const render_container_published_ports = (ports) => {
-        </DescriptionListGroup>
-        <DescriptionListGroup>
-            <DescriptionListTerm>{_("MAC address")}</DescriptionListTerm>
-            <DescriptionListDescription>{containerDetail ? containerDetail.NetworkSettings.MacAddress : ""}</DescriptionListDescription>
-        </DescriptionListGroup>
-    </DescriptionList>
+    <Flex spaceItems={{ modifier: 'spaceItemsXl' }}>
+        <DescriptionList className='container-details-basic'>
+            <DescriptionListGroup>
+                <DescriptionListTerm>{_("ID")}</DescriptionListTerm>
+                <DescriptionListDescription>{container.Id}</DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+                <DescriptionListTerm>{_("Image")}</DescriptionListTerm>
+                <DescriptionListDescription>{container.Image}</DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+                <DescriptionListTerm>{_("Command")}</DescriptionListTerm>
+                <DescriptionListDescription>{container.Command ? utils.quote_cmdline(container.Command) : ""}</DescriptionListDescription>
+            </DescriptionListGroup>
+        </DescriptionList>
+        <DescriptionList columnModifier={{ default: '2Col' }} className='container-details-networking'>
+            <DescriptionListGroup>
+                <DescriptionListTerm>{_("Ports")}</DescriptionListTerm>
+                <DescriptionListDescription>{render_container_published_ports(container.Ports)}</DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+                <DescriptionListTerm>{_("IP address")}</DescriptionListTerm>
+                <DescriptionListDescription>{containerDetail ? containerDetail.NetworkSettings.IPAddress : ""}</DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+                <DescriptionListTerm>{_("Gateway")}</DescriptionListTerm>
+                <DescriptionListDescription>{containerDetail ? containerDetail.NetworkSettings.Gateway : ""}</DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+                <DescriptionListTerm>{_("MAC address")}</DescriptionListTerm>
+                <DescriptionListDescription>{containerDetail ? containerDetail.NetworkSettings.MacAddress : ""}</DescriptionListDescription>
+            </DescriptionListGroup>
+        </DescriptionList>
+        <DescriptionList className='container-details-state'>
+            <DescriptionListGroup>
+                <DescriptionListTerm>{_("Created")}</DescriptionListTerm>
+                <DescriptionListDescription>{utils.localize_time(Date.parse(container.Created) / 1000)}</DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+                <DescriptionListTerm>{_("State")}</DescriptionListTerm>
```

Comment:
Do we need state here as well if we have it  the overview already?
Also, how come it shows something different here than in overview?
![Screenshot from 2021-09-08 09-33-10huh](https://user-images.githubusercontent.com/12330670/132466309-b95c2ea8-5869-41e3-8291-7b432f9fc666.png)

Reply:
edit - for running it makes sense, as it shows `Running` and in details `Up since <date>`